Project/Homebrew/README.md

```diff
@@ -0,0 +1,22 @@
+Building
+--------
+
+Install ffmpeg, freetype2, qt5
+
+    $ brew install ffmpeg freetype2 qt5
+
```

Comment:
should mention that sometimes `brew link --force qt5` is needed, since homebrew normally doesn't link qt5